source/braille.py

```diff
@@ -2228,13 +2272,24 @@ def formatCellsForLog(cells: List[int]) -> str:
 @type currentCellCount: bool
 """
 
-filter_displaySize = extensionPoints.Filter()
+filter_displaySize = extensionPoints.Filter[int]()
 """
 Filter that allows components or add-ons to change the display size used for braille output.
 For example, when a system has an 80 cell display, but is being controlled by a remote system with a 40 cell
 display, the display size should be lowered to 40 .
 @param value: the number of cells of the current display.
-@type value: int
+Note: filter_displayDimensions should now be used in place of this filter.
+If this filter is used, NVDA will assume that the display has 1 row of `displaySize` cells.
+"""
+
+DisplayDimensions = NamedTuple("DisplayDimensions", [("numRows", int), ("numCols", int)])
```

Comment:
Have you considered this syntax, which is a bit cleaner IMO:
```suggestion
class DisplayDimensions(NamedTuple):
	numRows: int
	numCols: int

	@property
	def displaySize(self):
		return self.numCols * self.numRows
```

Reply:
Thanks, I like that. I can Stop writing `numRows * numCols` everywhere.